include/grpcpp/security/tls_credentials_options.h

```diff
@@ -317,6 +321,11 @@ class TlsCredentialsOptions {
    * goes unused when creating channel credentials, and the user can set it to
    * GRPC_SSL_DONT_REQUEST_CLIENT_CERTIFICATE. **/
   grpc_ssl_client_certificate_request_type cert_request_type_;
+  /** The server_verification_option_ flag is only relevant when the
+   * TlsCredentialsOptions are used to instantiate client credentials; the flag
+   * goes unused when creating server credentials, and the user can set it to
+   * GRPC_SSL_SERVER_VERIFICATION. **/
```

Comment:
By `user`, do you mean `client`? Do you intend to say the default option is `GRPC_SSL_SERVER_OPTION`? This sentence seems a little redundant to me.